cluster-scope/overlays/nerc-ocp-infra/redhatcop.redhat.io/odf-node-patcher.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: Patch
+metadata:
+  name: odf-node-label-ctl-0
+  namespace: openshift-storage
+spec:
+  serviceAccountRef:
+    name: patcher
+  patches:
+    odf-node-label-ctl-0:
+      targetObjectRef:
+        apiVersion: v1
+        kind: Node
+        name: ctl-0
```

Comment:
I think that if we drop the `name` field from the `targetObjectRef`, the patch would apply to all node resources. There's a table in the [patch enforcement](https://github.com/redhat-cop/patch-operator#runtime-patch-enforcement) section of the README that suggests this.

Reply:
makes sense, I have made that change for the nerc-ocp-infra nodes.